engines/ic3bits.h

```diff
@@ -0,0 +1,51 @@
+/*********************                                                  */
+/*! \file ic3bits.h
+** \verbatim
+** Top contributors (to current version):
+**   Makai Mann
+** This file is part of the pono project.
+** Copyright (c) 2019 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief Bit-level IC3 implementation that splits bitvector variables
+**        into the individual bits for bit-level cubes/clauses
```

Comment:
Maybe it is worth to comment here that the algorithm (frames and cubes) work with bits but the transition system is still at the BV level

Reply:
Good idea! Added with d6cda362861b7f07abc9e8f6cf0a361bfe658501.